cupy/random/distributions.py

```diff
@@ -276,6 +276,33 @@ def normal(loc=0.0, scale=1.0, size=None, dtype=float):
     return x
 
 
+def poisson(lam=1.0, size=None, dtype=int):
+    """Poisson distribution.
+
+    Returns an array of samples drawn from the poisson distribution. Its
+    probability mass function is defined as
+
+    .. math::
+        f(x) = \\frac{\\lambda^xe^{-\\lambda}}{k!},
+
+    Args:
+        lam (array_like of floats): Parameter of the poisson distribution
+            :math:`\\lambda`.
+        size (int or tuple of ints): The shape of the array. If ``None``, a
+            zero-dimensional array is generated.
```

Comment:
The returned array is not zero-dimensional unless `lam` is.

Reply:
Oh, I didn't notice...
I'll also fix documents of the others in an other PR.